### PR TITLE
🚀 Report cls-fcp and cls-vis to compare with cls

### PR DESCRIPTION
**Pull request by rcebulko**
Data suggests some CLS violations are occurring either before FCP, before iframe visibility, or both. This PR adds reporting for two new metrics:
- **cls-fcp**: CLS at the time of the first contentful paint
- **cls-ofv**: CLS at the time the AMP doc becomes visible

### CODE REVIEW

**Review by kristoferbaxter — APPROVED** [review]
This looks good to me. Good to get another reviewer @ampproject/wg-performance?

File: src/service/performance-impl.js [hunk 1]
```diff
@@ -350,6 +355,13 @@ export class Performance {
         this.tickDelta(TickLabel.FIRST_CONTENTFUL_PAINT, value);
         this.tickSinceVisible(TickLabel.FIRST_CONTENTFUL_PAINT_VISIBLE, value);
         recordedFirstContentfulPaint = true;
+
+        // TODO(rcebulko): remove after data collection
```
Comment (erwinmombay) [inline src/service/performance-impl.js:359]: nit, add `#`

**Review by erwinmombay — APPROVED** [review]

File: src/service/performance-impl.js (hunk 1 above)
Comment (rcebulko) [reply src/service/performance-impl.js:359]:
```suggestion
        // TODO(#33207): remove after data collection
```

File: src/enums.js [hunk 2]
```diff
@@ -36,6 +36,9 @@ export const TickLabel = {
   CONTENT_LAYOUT_DELAY: 'cld',
   CUMULATIVE_LAYOUT_SHIFT: 'cls',
   CUMULATIVE_LAYOUT_SHIFT_2: 'cls-2',
+  // TODO(#33207): Remove after data collection
+  CUMULATIVE_LAYOUT_SHIFT_BEFORE_FCP: 'cls-fcp',
+  CUMULATIVE_LAYOUT_SHIFT_BEFORE_VISIBLE: 'cls-vis',
```
Comment (samouri) [inline src/enums.js:41]: super nit: 
```suggestion
  CUMULATIVE_LAYOUT_SHIFT_BEFORE_VISIBLE: 'cls-ofv',
```

**Review by samouri — APPROVED** [review]

**Review by kristoferbaxter — APPROVED** [review]
Approval for bundle-size.

**Comment by rcebulko** [comment]
@erwinmombay Thanks for pushing this across the finish line!

**Pull request merged**